sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/AbstractDistributeTest.java

```diff
@@ -61,27 +71,31 @@
 
 @SuppressWarnings("CanBeFinal")
 @PrepareForTest({
-        AppCenter.class,
-        AppCenterLog.class,
-        AppNameHelper.class,
-        BrowserUtils.class,
-        CryptoUtils.class,
-        Distribute.class,
-        HandlerUtils.class,
-        HttpUtils.class,
-        InstallerUtils.class,
-        NetworkStateHelper.class,
-        ReleaseDetails.class,
-        SharedPreferencesManager.class,
-        TextUtils.class,
-        Toast.class
+    AppCenter.class,
+    AppCenterLog.class,
+    AppNameHelper.class,
+    BrowserUtils.class,
+    CryptoUtils.class,
+    Distribute.class,
+    DistributeUtils.class,
+    HandlerUtils.class,
+    HttpUtils.class,
+    InstallerUtils.class,
+    NetworkStateHelper.class,
+    ReleaseDetails.class,
+    ReleaseDownloaderFactory.class,
+    SharedPreferencesManager.class,
+    TextUtils.class,
+    Toast.class
 })
 public class AbstractDistributeTest {
 
     static final String TEST_HASH = HashUtils.sha256("com.contoso:1.2.3:6");
 
     private static final String DISTRIBUTE_ENABLED_KEY = KEY_ENABLED + "_Distribute";
 
+    private static final String LOCAL_FILENAME_PATH_MOCK = "ANSWER_IS_42";
```

Comment:
```suggestion
    private static final String LOCAL_FILENAME_PATH_MOCK = "ANSWER_IS_42";
```
I see what you did there ;)